docs/topics/settings.txt

```diff
@@ -224,6 +224,7 @@ In this example, default settings are taken from ``myapp_defaults``, and the
 ``myapp_defaults``::
 
     from django.conf import settings
+
```

Comment:
Not sure if we will keep the line changes  in this file which are separate to the relative import changes

Reply:
I removed them